docs/src/ZIndex Classes.doc.js

```diff
@@ -67,10 +67,12 @@ card(
 Export FixedZIndex and CompositeZIndex rather than the z-index values itself.
 ~~~jsx
 // exporting file
+import { FixedZIndex } from 'gestalt';
+
 export const BaseZIndex = = new FixedZIndex(1);
 
 // importing file
-import { BaseZIndex } from 'gestalt';
+import { BaseZIndex } from 'file';
```

Comment:
This makes it seem like the import is from [this npm package](https://www.npmjs.com/package/File)…consider something like `'./path/to/your/zindex/file.js'`

Reply:
DONE

Reply:
I don't think you committed that change? Or you didn't push it up?

Reply:
mmm i did, maybe didn't push.

Reply:
maybe i didn't///